app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -130,7 +141,14 @@ class GithubWebhook extends RequestHandler<Body> {
     }
     // The mergeable flag may be null. False indicates there's a merge conflict,
     // null indicates unknown. Err on the side of allowing the job to run.
-    if (event.pullRequest.mergeable != false && await _checkForCqLabel(labels)) {
+
+    // We flutter/flutter tests need to be optimized before enforcing CQ.
+    bool runCQ = true;
+    if (needsCQLabelList.contains(event.repository.fullName.toLowerCase())) {
+      runCQ = await _checkForCqLabel(labels);
+    }
+
+    if (event.pullRequest.mergeable != false && runCQ) {
```

Comment:
I think the `mergeable` should probably be removed - it's not trustworthy based on other experience.

Reply:
Done!